src/main/java/org/jabref/gui/search/GlobalSearchBar.java

```diff
@@ -224,7 +223,7 @@ private void initSearchModifierButtons() {
         fulltextButton.setSelected(searchPreferences.isFulltext());
         fulltextButton.setTooltip(new Tooltip(Localization.lang("Fulltext search")));
         initSearchModifierButton(fulltextButton);
-        EasyBind.subscribe(preferencesService.getFilePreferences().fulltextIndexLinkedFilesProperty(), value -> fulltextButton.setDisable(!value));
+        fulltextButton.disableProperty().bind(preferencesService.getFilePreferences().fulltextIndexLinkedFilesProperty().not());
```

Comment:
Still Needs to be bound to the UI button state and maybe we can bind managed (completely removed from the UI) to the fulltextIndexLinkedFilesProperty property